src/clipboardobserver.js

```diff
@@ -22,17 +22,53 @@ export default class ClipboardObserver extends DomEventObserver {
 	constructor( doc ) {
 		super( doc );
 
-		this.domEventType = [ 'paste', 'copy', 'cut' ];
+		this.domEventType = [ 'paste', 'copy', 'cut', 'drop' ];
+
+		doc.on( 'paste', _handleInput, { priority: 'low' } );
```

Comment:
This is unsafe and incorrect. `this.listenTo()` should be used. Those listeners should be gone together with the observer.